tests/console/system_prepare.pm

```diff
@@ -131,6 +133,17 @@ sub run {
         my $regcode = (get_var('AGAMA_PRODUCT_ID') =~ /SAP/) ? get_var('SCC_REGCODE_SLES4SAP') : get_var('SCC_REGCODE');
         my $regurl = get_var('SCC_URL');
         assert_script_run("suseconnect -r $regcode --url $regurl");
+
+        # Register HA extension and output some debug info for reference
+        if (get_var('SCC_ADDONS', '') eq 'ha') {
```

Comment:
I believe this is OK for the time being, but usually `SCC_ADDONS` can contain a comma-separated list of addons, so this will probably need to be changed if other addons are available.

Reply:
Yes. 
I added `HA workaround` between bellow lines. So the whole temporary workaround will be deleted as soon as agama support HA installation. So I did not handle `comma-separated list of addons`.

```
    ########
    # Workaround to install sle16 in beta1
...
if (get_var('SCC_ADDONS', '') eq 'ha') {
}
...
    ########
```